### PR TITLE
fix: alignment in homepage

### PR DESCRIPTION
**Pull request by simboonlong**
## Before
before

## After
after

- alignment consistency fix
- mainly rectified the dropzone box-shadow as "padding", as box-shadow is not taking up space
- replacing myjson with gist sample to fix load-action e2e test

### CODE REVIEW

File: src/components/HomePageContent/AboutSection.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const HowItWorks = () => {
 
 const AboutSection = () => (
   <section className={`bg-light ${css["section"]}`} id="how-it-works">
-    <div className="container">
+    <div className="container-custom">
```
Comment (Nebulis) [inline src/components/HomePageContent/AboutSection.tsx:42]: container-custom ? is it to skip bootstrap ?

File: src/test/load-action-certificate.spec.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { Selector } from "testcafe";
 
-fixture("Load action from plain certificate").page`http://localhost:3000`;
+fixture.skip("Load action from plain certificate").page`http://localhost:3000`;
```
Comment (Nebulis) [inline src/test/load-action-certificate.spec.js]: of course you fix this in the next MR :D

Comment (Nebulis) [reply src/test/load-action-certificate.spec.js]: ctually I prefer the opposite, fix the test first then merge this MR :)

Comment (simboonlong) [reply src/test/load-action-certificate.spec.js]: ok fixed by replacing a gist sample instead of myjson

**Review by Nebulis — APPROVED** [review]

**Pull request merged**